core/vm/instructions.go

```diff
@@ -817,6 +818,12 @@ func opStop(pc *uint64, interpreter *EVMInterpreter, scope *ScopeContext) ([]byt
 }
 
 func opSelfdestruct(pc *uint64, interpreter *EVMInterpreter, scope *ScopeContext) ([]byte, error) {
+	// [Kroma: START]
```

Comment:
Currently there is no disable for the Selfdestruct Op. Will this code be disabled from the time it is applied to the sequencer until MPT time?